components/dashboard/InsightsPanel.tsx

```
@@ -111,447 +80,325 @@ function InsightsPanelContent({
           if (!result.success) {
-            throw new Error(result.message || 'Action execution failed');
+            throw new Error(result.message || 'اجرای action شکست خورد');
           }
           if (result.redirectUrl) {
-            announce(`Navigating to ${result.redirectUrl}`);
+            announce(`در حال انتقال به ${result.redirectUrl}`);
           }
           break;
         case 'external':
-          // Validate external URL
           try {
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)